### PR TITLE
[IOCOM-1128] Update scaling rules of messages functions

### PR DESCRIPTION
**Pull request by lucacavallaro**
### List of changes
Update scaling rules for `app-messages`, `messages_cqrs` and `pushnotif` functions.

### Motivation and context
Improving the scalability responsiveness of the function
(Same as https://github.com/pagopa/io-infra/pull/890)

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [x] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-platform-iac-projects](https://dev.azure.com/pagopaspa/io-platform-iac-projects/_build?view=folders)
2. select PR branch
3. wait for approval

### CODE REVIEW

File: src/domains/messages-app/07_function_pushnotif.tf [hunk 1]
```diff
@@ -290,7 +290,7 @@ resource "azurerm_monitor_autoscale_setting" "push_notif_function" {
         direction = "Decrease"
         type      = "ChangeCount"
         value     = "1"
-        cooldown  = "PT20M"
+        cooldown  = "PT10M"
```
Comment (balanza) [inline src/domains/messages-app/07_function_pushnotif.tf:293]: **thought:** ‏It took me a bit to get how to read those rules, as I've been far from these configurations for some time. Also, Azure docs are a bit misaligned with terraform API: https://learn.microsoft.com/en-us/azure/azure-monitor/autoscale/autoscale-understanding-settings.

As far as I get it, it reads as _"scale up or down based on the average number of requests in the last 15 minutes; add or remove an instance every 10 minutes"_. Am I right?

I advocate adding a comment above each rule, to explain it in plain English. So that readers can grasp the intended behavior.

**Review by balanza — DISMISSED** [review]
LGTM 

I just advocate for more comments.

**Review by Garma00 — DISMISSED** [review]

**Review by michaeldisaro — DISMISSED** [review]
LGTM.

**Review by michaeldisaro — APPROVED** [review]
